src/actions/write/setRecord.test.ts

```diff
@@ -25,18 +25,27 @@ import { baselineAntState } from '../../../tests/utils/constants';
 import { setRecord } from './setRecord';
 
 describe('setRecord', () => {
+  let state = { ...baselineAntState };
+
+  beforeEach(() => {
+    state = { ...baselineAntState };
+  });
```

Comment:
```suggestion
```

Reply:
removed - thought this was needed when trying to debug a test. Misunderstood how padEnd worked 😓